### PR TITLE
Reactored settings view

### PR DESCRIPTION
**Pull request by linjiang82**
**Problem:**
The settings page was not in line with design docs.
**Solution:**
Refactored the UI of the settings page.
Functions of each setting were not implemented.

### CODE REVIEW

**Comment by connorhsm** [comment]
Fixes #188

File: lib/ui/views/settings_view.dart [hunk 1]
```diff
@@ -44,130 +44,340 @@ class _SettingsPageState extends State<SettingsPage> {
...
+              "Account",
+              style: TextStyle(
+                  color: Theme.of(context).colorScheme.primary,
+                  fontSize: 14,
+                  fontWeight: FontWeight.w700),
+            ),
+          ),
+        ),
+        SizedBox(
+          height: 30,
```
Comment (oscarmanderj) [inline lib/ui/views/settings_view.dart:114]: This SizedBox pattern doesn't really work - The click event isn't on top of the actual arrow, as the sized box is restricting the height of the list tile. Can you look for a different solution. Either remove the height or use a different widget.

**Review by oscarmanderj — COMMENTED** [review]
Other than comment - Everything else good

File: lib/ui/views/settings_view.dart (hunk 1 above)
Comment (linjiang82) [reply lib/ui/views/settings_view.dart:114]: No problem, I will see how to solve it tonight

**Review by KipCrossing — COMMENTED** [review]
I assume this PR is purely cosmetic

**Comment by oscarmanderj** [comment]
Yes - Cosmetic. Let's do all the setting's pages in a new PRs progressively

File: lib/ui/views/settings_view.dart (hunk 1 above)
Comment (linjiang82) [reply lib/ui/views/settings_view.dart]: Hi mate, could you please check the modification and see if that is ok now?

**Review by KipCrossing — APPROVED** [review]
LGTM

**Pull request merged**